src/main/kotlin/no/digdir/servicecatalog/service/PublicServiceService.kt

```diff
@@ -89,4 +91,29 @@ class PublicServiceService(private val publicServiceRepository: PublicServiceRep
         findPublicServiceById(id, catalogId)
             ?.takeIf { it.published }
             ?: throw CustomNotFoundException()
+
```

Comment:
flytt alt av count-kode i en egen service som tar for seg både tjenester og offentlig tjenester samtidig